### ISSUE TITLE
cant get manifest to run

### ISSUE THREAD

**Issue by drizzt09** [akarazniewicz/stremio-jellyfin#5]
i have Ubuntu installed on AWS Free Tier server.
I have installed Jellyfin and stremio-jellyfin on AWS.
Jellyfin is up (no library yet).

repo is in JF

It says im all connected in logs:
![image](https://github.com/akarazniewicz/stremio-jellyfin/assets/6366965/e30ad0eb-ad28-4df3-bd32-ac8da6900b73)

but stremio fails to connect to the manifest:
![image](https://github.com/akarazniewicz/stremio-jellyfin/assets/6366965/d2471060-08c4-4281-9e6c-eb830194a230)

i have tried the URL in first picture, but that doesnt make sense. tried with aws public IP https and http
i have tried my URL https and http (but http auto forces https).

All give same error.

**Comment by drizzt09** [comment]
@akarazniewicz 
so i found the problem. there is no "manifest.json" file.
There is a manifest.js. js not json

I tried copying manifest.js to manifest.json but still get same failed to fetch error even though I can open it with same URL in browser

**Comment by zivavitan** [comment]
Same issue here... have you able to fix it ?

**Comment by drizzt09** [comment]
> Same issue here... have you able to fix it ?

Yeah the dev should just archive it if he isn't going to fix or update the code or even respond to people commenting.
